app/src/main/java/com/wednesday/bankingdetails/adapter/BankListAdapter.kt

```diff
@@ -17,10 +16,15 @@ class BankListAdapter(var bankDataSource: ArrayList<Bank>) :
     class ViewHolder(override val containerView: View) : RecyclerView.ViewHolder(containerView),
         LayoutContainer {
 
+        private fun String.capitalizeFirstLetter(): String =
```

Comment:
Create a new folder called `extensions` and add this code there. Remember in kotlin functions are first class members. So you can create a kotlin file and add this code there so it has global namespace